src/index.html

```diff
@@ -77,7 +79,13 @@ <h1>Stephen Gurney</h1>
       </p>
```

Comment:
_generally_ we would use `<br>` tags very rarely - they are good for adding a line break between items, but there are usually other preferable ways to achieve this. There are basically two reasons for breaks to appear between/within tags:

- it makes semantic sense
- to style the UI

Starting with the first one: it makes semantic sense. There aren't too many reasons where a `<br>` is the better choice over something else. For example, if we are breaking up multiple paragraphs of text, we could instead wrap each paragraph in it's own set of `<p>` tags rather than a single `<p>` tag with `<br>` between each paragraph of text (this is what you had previously it looks like 🙂).

If we take a look at the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br) it gives the example of an address - this is a fair example since _semantically_ each line in an address is not a separate paragraph, but they are separate to each other (and in normal writing these would be split into different lines). Generally though, you'll find you don't come across many situations where you need a single paragraph broken up by lines for semantic reasons.

This takes us on to the second reason: to style the UI. For this we should always prefer to use CSS, since the reason we want to break items up is for visual reasons rather than semantic ones (ie. we use HTML to write a semantically valid document without caring what it looks like, and defer to using CSS to make it look good).

So whilst we could use a `<br>` to apply some spacing between two elements (for example, you've put a `<br>` betwen a heading tag and a paragraph tag which would create some spacing), we should instead prefer applying styles to one of these elements to create the spacing between them (ie. apply margin to the bottom of the heading or apply margin to the top of the paragraph). In addition to this being a good separation of concerns (HTML for semantic documents, CSS for styling) it also gives us the benefit of much greater control - we can very easily set that margin to any value we like for just that heading via a CSS class, rather than relying on the in build browsers styles which determine how much spacing the `<br>` tag will give us

Reply:
Ahh awesome, thanks for that. I will revert it to how I had it originally and add margins instead of using the <br> tag.